DBMS/Assignment1/library.sql

```diff
@@ -0,0 +1,128 @@
+/* created database*/
+CREATE DATABASE Library_Management;
+USE Library_Management;
+
+/* created table named members*/
+CREATE TABLE members(
+member_id INT PRIMARY KEY AUTO_INCREMENT,
+member_nm VARCHAR(20),
+addressline1 VARCHAR(20),
+addressline2 VARCHAR(20),
+category CHAR);
+
+/* created table named authors*/
+CREATE TABLE authors(
```

Comment:
ACTION: Table Name should be "author". Tables, views, and other relations that hold data should have singular names, not plural.